cfg/Tomb1Main.json5

```diff
@@ -24,6 +24,9 @@
     // Draws flame when firing a shotgun, like for other guns.
     "enable_shotgun_flash": true,
 
+    // Shotgun no longer fires when a locked target moves out of Lara's sight, like for other guns.
+    "enable_shotgun_target_fix": false,
```

Comment:
Should default to true here.

Reply:
Ah sorry. Fixed.

Reply:
Fix won't be here until you push it, so I'm not resolving this one yet.